src/lava/magma/core/learning/constants.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2021-22 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import lava.magma.core.learning.string_symbols as str_symbols
+
+# ---------------------------------------------------------------------------
+# Width constants (only for fixed-point implementation)
+
+W_WEIGHTS_U = 8
```

Comment:
All these constants could use some comments that describe what they are. Maybe an explanatory comment for all of them (Number of bits used for each of these fields ?) and for each group?

Reply:
done